tests/helpers/shared-test-setups/ember-container-query.ts

```diff
@@ -18,7 +18,6 @@ const context: Context = {
     ['tracks', new Set(['.d.ts', '.hbs'])],
     ['tracks/list', new Set(['.hbs', '.ts'])],
     ['tracks/table', new Set(['.d.ts', '.hbs'])],
-    ['types', new Set(['.ts'])],
```

Comment:
The removal of `'types'` is expected, as `components/types.ts` is not really a component file.